spec/support/seeds/seeds_e2e.rb

```diff
@@ -93,6 +93,24 @@
   user.profile.update(website_url: Faker::Internet.url)
 
   user.add_role(:trusted)
+
+  seeder.create_if_doesnt_exist(Article, "slug", "unfeatured-article-slug") do
```

Comment:
In Cypress, we did not have any unfeatured articles, so I created one in the seeds.